tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -316,7 +316,7 @@ module.exports = {
   },
   elements: {
     filesTable: {
-      selector: '#files-list'
+      selector: '#files-list, #shared-with-list'
```

Comment:
what is this doing? how is that working with others file lists?

Reply:
That is basically `OR` operator. It works when it gets either of those ids. The `shared with pages` has different id than the main files page. With this we can use the already defined steps on the `shared with me` page too. But we have to decide weather to use this or not.